js/blocks/loader/edit.js

```diff
@@ -34,14 +33,22 @@ const formControls = ( props, block ) => {
 }
 
 const previewData = ( props, block ) => {
-	if ( typeof props.attributes.block_template !== 'undefined' && props.attributes.block_template.length > 0 ) {
```

Comment:
I believe this line was intended to prevent the preview from updating when the block was blurred, but since we're no longer updating with every single keystroke and change in the field settings, I removed the line.

Reply:
I could be wrong about its purpose, though. In the end, this code doesn't work if this line exists.